experimental/traffic-portal/.eslintrc.json

```diff
@@ -349,7 +349,7 @@
 						],
 						"require": {
 							"ClassDeclaration": true,
-							"MethodDefinition": true
+							"MethodDefinition": false
```

Comment:
The rules shouldn't need to change to fix this. Are there unreported failures currently committed? In that case, just let the linter fail and we can fix them in a follow-up.

Reply:
> The rules shouldn't need to change to fix this. Are there unreported failures currently committed? In that case, just let the linter fail and we can fix them in a follow-up.

currently, when you run lint, then the linter will not show the methods that don't have jsdoc. I will revert this change, and you'll see the lint fail

Reply:
That's fine. It even helps demonstrate that your PR does what it should be doing. Once this is merged I'll go through and fix the lint errors.